task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -74,7 +74,7 @@ spec:
         echo "Scanning image. This operation may take a while."
         clamscan -ri --max-scansize=4095M --max-filesize=4095M \
           --max-scantime=0 --max-files=0 --max-recursion=1000 --max-dir-recursion=20000 --max-embeddedpe=4095M \
-          --max-htmlnormalize=4095M --max-htmlnotags=4095M --max-scriptnormalize=4095M --max-ziptypercg=4095M \
```

Comment:
by removing it, we are basically getting the default which is 

```
       --max-htmlnormalize=#n
              Maximum size of HTML file to normalize. You may pass the value in kilobytes in format xK or xk, or megabytes in format xM or xm, where x is a number (default: 10 MB, max: <4 GB).
```

Do we want to rather change it to 10M instead of removing?

Reply:
I would say we could try cutting it in half and not have the upper limit set as we do now those 4 gigs.

Reply:
Unfortunately, 2 gigs will cause issues for the component I looked at. 10M should be safe.

1 gig as well, for this file I looked at. Not sure if there will be other inputs where we'll see problems with 1 gig.